tests/conftest.py

```diff
@@ -10,12 +11,62 @@
 import anndata as ad
 from anndata.tests.helpers import subset_func  # noqa: F401
 
+if TYPE_CHECKING:
+    from types import EllipsisType
+
 
 @pytest.fixture
 def backing_h5ad(tmp_path):
     return tmp_path / "test.h5ad"
 
 
+@pytest.fixture(
+    params=[
+        pytest.param(((...), (slice(None), slice(None))), id="ellipsis"),
```

Comment:
`(...)` isn’t a tuple, maybe you wanted it to be one?

Reply:
hmmm true, yes, the return type should just include this then in a union.  thanks!